### PR TITLE
Feat: Add Push to W3I

### PR DESCRIPTION
**Pull request by Elyniss**

### CODE REVIEW

File: samples/wallet/src/main/kotlin/com/walletconnect/sample/wallet/domain/PushWalletDelegate.kt [hunk 1]
```diff
@@ -17,18 +18,25 @@ object PushWalletDelegate: PushWalletClient.Delegate {
...
+        Log.d("W3W PW", "onPushMessage()")
+
+//        scope.launch { _wcPushEventModels.emit(pushMessage) }
     }
 
     override fun onPushDelete(pushDelete: Push.Wallet.Event.Delete) {
-        scope.launch { _wcPushEventModels.emit(pushDelete) }
+        Log.d("W3W PW", "onPushDelete()")
+
+//        scope.launch { _wcPushEventModels.emit(pushDelete) }
```
Comment (jakubuid) [inline samples/wallet/src/main/kotlin/com/walletconnect/sample/wallet/domain/PushWalletDelegate.kt:39]: Do you want to keep all of those logs? let's remove commented code

File: web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/webview/WebViewPresenter.kt [hunk 2]
```diff
@@ -44,18 +44,21 @@ internal class WebViewPresenter(
...
             WEB3INBOX_AUTH_PROVIDER_TYPE_KEY to WEB3INBOX_AUTH_PROVIDER_TYPE_VALUE,
+            WEB3INBOX_PUSH_PROVIDER_TYPE_KEY to WEB3INBOX_PROVIDER_TYPE_VALUE,
             WEB3INBOX_ACCOUNT_KEY to accountId.address()
         )
     )
 
     internal companion object {
-        const val WEB3INBOX_URL = "https://web3inbox-dev-hidden.vercel.app"
-        const val WEB3INBOX_PROVIDER_TYPE_KEY = "chatProvider"
+//        const val WEB3INBOX_URL = "https://web3inbox-dev-hidden.vercel.app"
```
Comment (jakubuid) [inline web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/webview/WebViewPresenter.kt:55]: Remove?

**Review by jakubuid — APPROVED** [review]

File: web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/json_rpc/Web3InboxRPCCallPushUpdateJsonAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,148 @@
+// Code generated by moshi-kotlin-codegen. Do not edit.
```
Comment (TalhaAli00) [inline web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/json_rpc/Web3InboxRPCCallPushUpdateJsonAdapter.kt:1]: let's remove this line

File: web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/json_rpc/Web3InboxRPCCallPushSubscriptionJsonAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,148 @@
+// Code generated by moshi-kotlin-codegen. Do not edit.
```
Comment (TalhaAli00) [inline web3/inbox/src/main/kotlin/com/walletconnect/web3/inbox/json_rpc/Web3InboxRPCCallPushSubscriptionJsonAdapter.kt:1]: let's remove this

**Pull request merged**